community-operators/svt-app-operator/svt-app-operator.v0.0.8.clusterserviceversion.yaml

```diff
@@ -0,0 +1,244 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: svt-app-operator.v0.0.8
+  namespace: placeholder
+  annotations:
+    capabilities: Seamless Upgrades
+    categories: "OpenShift Optional"
+    certified: "false"
+    description: An operator to run the SVT app
+    containerImage: docker.io/hongkailiu/svt-app-operator:a012
+    createdAt: 2019-01-01T11:59:59Z
+    support: RH SVT
+spec:
+  version: 0.0.8
+  displayName: SVT Application
+  provider:
+    name: RH SVT
+    url: www.redhat.com
+  maintainers:
+    - email: svt@redhat.com
+      name: RH SVT
+  maturity: alpha
+  labels:
+    olm-owner-enterprise-app: svt-app-operator
+    olm-status-descriptors: svt-app-operator.v0.0.8
+  description: SVT Tool in GoLang
```

Comment:
This can be a longer description for your operator. This should include a description of what the operator does and any manual steps required during setup

Reply:
See my comment below.

Reply:
I noticed the `metadata.annotations.description` is more descriptive than the `spec.description`, so let me clarify the purpose of each one.
`metadata.annotations.description` should be a short description, recommended less than 135 characters. In operatorhub.io this will be the description displayed in the operator tile as a preview description.
`spec.description` is the long description for your operator, with no length limit. This description can be multiple lines long, and supports markdown. In operatorhub.io this is the description shown after you click on the operator, bringing you to a page with info about the operator.
Does this help? Feel free to ask any questions. I suggest checking out operatorhub.io to see the descriptions for other operators.

Reply:
Did not notice there were 2 of them and I was changing the wrong one.
Sorry for that.
See if it looks ok now.